iron-a11y-announcer.html

```diff
@@ -57,68 +57,67 @@
   </template>
 
   <script>
-
     (function() {
       'use strict';
 
-      Polymer.IronA11yAnnouncer = Polymer({
-        is: 'iron-a11y-announcer',
-
-        properties: {
-
-          /**
-           * The value of mode is used to set the `aria-live` attribute
-           * for the element that will be announced. Valid values are: `off`,
-           * `polite` and `assertive`.
-           */
-          mode: {
-            type: String,
-            value: 'polite'
-          },
-
-          _text: {
-            type: String,
-            value: ''
+      class IronA11yAnnouncer extends Polymer.Element {
+        static get is() { return 'iron-a11y-announcer'; }
+        static get properties() {
+          return {
+
+            /**
+             * The value of mode is used to set the `aria-live` attribute
+             * for the element that will be announced. Valid values are: `off`,
+             * `polite` and `assertive`.
+             */
+            mode: {
+              type: String,
+              value: 'polite'
+            },
+
+            _text: {
+              type: String,
+              value: ''
+            }
+          };
+        }
+        static requestAvailability() {
+          if (!Polymer.IronA11yAnnouncer.instance) {
+            Polymer.IronA11yAnnouncer.instance = new Polymer.IronA11yAnnouncer();
           }
-        },
 
-        created: function() {
+          document.body.appendChild(Polymer.IronA11yAnnouncer.instance);
+        }
+
+        constructor() {
+          super();
           if (!Polymer.IronA11yAnnouncer.instance) {
             Polymer.IronA11yAnnouncer.instance = this;
           }
 
-          document.body.addEventListener('iron-announce', this._onIronAnnounce.bind(this));
-        },
+          document.body.addEventListener('iron-announce', (event) => {
+            if (event.detail && event.detail.text) {
+              this.announce(event.detail.text);
+            }
+          });
+        }
 
         /**
          * Cause a text string to be announced by screen readers.
          *
          * @param {string} text The text that should be announced.
          */
-        announce: function(text) {
+        announce(text) {
           this._text = '';
-          this.async(function() {
+          this.async(() => {
```

Comment:
async doesn't work when using Polymer.Element.
You need to add `<link rel="import" href="../polymer/lib/utils/async.html">`

And then replace it with:
```
Polymer.Async.microTask.run(function() {
            this._text = text;
          }.bind(this), 100)
```